### ISSUE TITLE
MNISTSuperpixel Experiment

### ISSUE THREAD

**Issue by HaniItani** [DeepMETv2/deepmetv2#18]
Hello,

I am interested in replicating your experiments on MNISTSuperpixels dataset. Can you please provide the training script for that? I tried DynamicReductionNetwork with input dim =1, hidden_dim=20, output_dim=num_classes=10, k=4, I used AdamW with base learning rate of 0.001 with cosine annealing, but the performance is very poor. Can you please advise?

Thank you very much for your time.
Best regards,
Hani

**Comment by lgray** [comment]
Hi - thanks for your interest! You can find the exact training script used in the paper on the arxiv:
https://arxiv.org/abs/2003.08013

Click "Download" -> "Other Formats" -> "gzipped archive", then within the downloaded tar file you'll find mnist.py which is the training script.

**Comment by lgray** [comment]
Your problem with performance is likely that you're not including the coordinates in your list of inputs (as you have input dim = 1). Without the coordinates the network will not be able to deduce practically anything.

**Comment by HaniItani** [comment]
Many thanks for the prompt reply. I figured the problem of appending the coordinates after a more careful read of your paper. Thank you very much for sharing the code!
